package.json

```diff
@@ -27,6 +27,7 @@
     "infusion": "3.0.0-dev.20170830T182157Z.392b2f8",
     "gpii-windows": "git://github.com/GPII/windows.git#hst-2017",
     "node-jqunit": "1.1.7",
-    "electron-edge": "6.5.4"
+    "electron-edge": "6.5.4",
+    "jquery": "^3.2.1"
```

Comment:
Was the incorrect version of jquery being included? I'm wondering if this might be a larger issue.

Reply:
@michelled, I think that jQuery was not included as a dependency at all. We need it in the PCP window for subsequent tasks in order to be able to perform DOM manipulations.

Reply:
@georgitodorov Infusion bundles jQuery with it by default

Reply:
@michelled, thank you, I removed the dependecy from the package.json and linked to the jQuery supplied together with infusion.